src/loader/level_loader.cpp

```diff
@@ -306,7 +307,7 @@ LevelData loadLevel(
   for (size_t i=0; i<header.numActorWords/3; ++i) {
     const auto type = levelReader.readU16();
     const base::Vector position{levelReader.readU16(), levelReader.readU16()};
-    actors.emplace_back(LevelData::Actor{position, type, std::nullopt});
+    actors.emplace_back(LevelData::Actor{position, static_cast<ActorID>(type), std::nullopt});
```

Comment:
Not exactly sure, what happens here, if a loaded value is not defined in the enum class.

Reply:
We could add some verification code that throws if it's not a known actor value. But that's maybe something for a different PR. At the moment, I guess it would store an invalid value in the enum value, and then fail later if t tries to load a Sprite?